sdk/servicebus/service-bus/src/clientEntityContext.ts

```diff
@@ -247,7 +250,7 @@ export namespace ClientEntityContext {
             connectionId,
             streamingReceiver.name
           );
-          await streamingReceiver.onDetached(error);
+          await streamingReceiver.onDetached(error, connectionDidDisconnect);
```

Comment:
onDetached() on the client entity context is only ever called from connection context i.e when disconnect event is fired

```suggestion
          await streamingReceiver.onDetached(error, true);
```

Reply:
Updated!